File: engine/model/src/main/scala/tools/packet/shop/RemoteChannelChange.scala
package tools.packet.shop

import net.opcodes.SendOpcode
import tools.packet.PacketInput

class RemoteChannelChange(private var _channelId: Byte) extends PacketInput {
  def channelId: Byte = _channelId

  override def opcode(): SendOpcode = SendOpcode.ENTRUSTED_SHOP_CHECK_RESULT
}

File: engine/engine-model/src/main/java/tools/packet/shop/RemoteChannelChange.java
package tools.packet.shop;

import net.opcodes.SendOpcode;
import tools.packet.PacketInput;

public record RemoteChannelChange(Byte channelId) implements PacketInput {
   @Override
   public SendOpcode opcode() {
      return SendOpcode.ENTRUSTED_SHOP_CHECK_RESULT;
   }
}